src/lwc/gsApplicationStatus/gsApplicationStatus.html

```diff
@@ -1,51 +1,44 @@
-<template >
+<template>
     <template if:false={isActiveInstance}>
-        <lightning-card>
+        <div class="container">
```

Comment:
Why go back to the non base component structure?

Reply:
Design needed a shadow which was not able to make it using a lightning-card